src/components/Glossary/GlossaryDefinition/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import { Box, Heading, Text } from "@chakra-ui/react"
+
+import Translation from "../../Translation"
+
+interface IProps {
+  term: string
+  tooltip?: boolean
```

Comment:
nitpick: for DX purposes, would rename this prop to something decoupled from implementation references and that express styling purposes. We could call this something like `size` which I think it is basically what we are doing here

```suggestion
  size?: "md" | "sm"
```

And then default it to `md`
```tsx
const GlossaryDefinition: React.FC<IProps> = ({ term, size = "md" }) => {
```

From the consumer's side, you would do
```tsx
<Tooltip content={<GlossaryDefinition term={hash!} size="sm" />}>
```

Now, other devs could use GlossaryDefinition in a more natural and easy to understand way.